app/models/measure.rb

```diff
@@ -73,6 +73,6 @@ def additional_code_sort
   end
 
   def key
```

Comment:
we can probably remove the method altogether right

Reply:
yeah, can be removed